implement autoslide and tests

cbikhazi opened the pull request:
New feature:  Able to implement an autoslide so that the slider automatically slides.  If property autoslide is set in milliseconds then it will autoslide otherwise the component will wait to UI.

erichbehrens commented on package.json:
```diff
@@ -33,6 +33,7 @@
   },
   "homepage": "https://github.com/erichbehrens/react-animated-slider#readme",
   "dependencies": {
+    "path": "^0.12.7",
```
Where is this used?

cbikhazi replied:
its not and it doesn't appear on my branch (not sure where it came from!), so if you want to remove then feel free.

erichbehrens commented:
Thanks for your submission. I've implemented a few more things in #28 

- disable timer on mouseover/touch: don't you hate it when something slides away from under your mouse pointer? :D
- set timer recursively

Please note I've also changed `autoslide` prop to `autoplay` (more consistent with html's `<video />` element `autoplay` attribute.

cbikhazi commented:
Hi,

Great stuff 😊

Have you updated the npm package?  I’d prefer to have a reference to your master code in the project I’m using the slider?

Thanks

Charles

From: Erich Behrens <notifications@github.com>
Sent: 06 March 2018 20:21
To: erichbehrens/react-animated-slider <react-animated-slider@noreply.github.com>
Cc: cbikhazi <cbikhazi@hotmail.com>; Author <author@noreply.github.com>
Subject: Re: [erichbehrens/react-animated-slider] implement autoslide and tests (#27)


Thanks for your submission. I've implemented a few more things in #28<https://nam01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Ferichbehrens%2Freact-animated-slider%2Fpull%2F28&data=02%7C01%7C%7Ce490abea30014c5e55d408d5839fccc8%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636559644711410367&sdata=DnPB1hSjt9m7nOfaxkb7%2B73qqJ%2BxGucPP7OutoOsoss%3D&reserved=0>

  *   disable timer on mouseover/touch: don't you hate it when something slides away from under your mouse pointer? :D
  *   set timer recursively

Please note I've also changed autoslide prop to autoplay (more consistent with html's <video /> element autoplay attribute.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://nam01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Ferichbehrens%2Freact-animated-slider%2Fpull%2F27%23issuecomment-370914806&data=02%7C01%7C%7Ce490abea30014c5e55d408d5839fccc8%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636559644711410367&sdata=RHC0EBZTfV8Ll%2BpzWNGVkVYkksFbvvYoFuI%2F%2BUMIqyg%3D&reserved=0>, or mute the thread<https://nam01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FACzG9sNYpTx3UBZuVEyyXMGKkI6ayKFZks5tbu-0gaJpZM4SbYQB&data=02%7C01%7C%7Ce490abea30014c5e55d408d5839fccc8%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636559644711410367&sdata=qEUGWkSybtYudwS1WTHqEKSavGY6TKr06llkrOnuhik%3D&reserved=0>.
